backend/apps/github/index/issue.py

```
@@ -78,11 +78,21 @@ class IssueIndex(IndexBase):
+            None
+        """
         return IndexBase.reindex_synonyms("github", "issues")
 
     def get_entities(self):
-        """Get entities for indexing."""
+        """
+        Get entities for indexing.
+
+        Returns:
+            QuerySet: A queryset of Issue objects to be indexed.
+        """
```

Ruff (0.8.2)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)